tubular/mapping.py

```diff
@@ -60,7 +65,8 @@ def __init__(self, mappings: dict[str, dict], **kwargs: dict[str, bool]) -> None
 
         super().__init__(columns=columns, **kwargs)
 
-    def transform(self, X: pd.DataFrame) -> pd.DataFrame:
+    @nw.narwhalify
+    def transform(self, X: FrameT) -> FrameT:
```

Comment:
Q: should we update the docstring to indicate this transformer can now accept polars dataframes?